src/lib/components/HttpErrors/HttpError.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Grid, Button, Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import Overridable from 'react-overridable';
+
+class HttpError extends Component {
+  render() {
+    const { title, message, icon, isBackOffice } = this.props;
+    return (
+      <Overridable id="HttpError.layout" {...this.props}>
+        <div className="frontsite">
+          <Grid
+            container
+            verticalAlign="middle"
+            textAlign="center"
+            className="error-page"
+          >
+            <Grid.Column>
+              <Icon name={icon} size="massive" />
+              <h1>{title}</h1>
+              <h3>{message}</h3>
+              {isBackOffice || (
```

Comment:
I am not sure what we are trying to achieve here, shouldn't the link `Back home` be displayed in every case? On a side note I believe for conditional rendering we need `&&` not `||`.

Reply:
I changed it because it didn't make sense to have a link "Back home" when the page was displayed in the backoffice. I can also keep the button and change the link to the backoffice's homepage.